ext_emconf.php

```diff
@@ -8,7 +8,7 @@
 # writing. "version" and "dependencies" must not be touched!
 ########################################################################
 
-$EM_CONF[$_EXTKEY] = [
+$EM_CONF['update_refindex'] = [
```

Comment:
Please let it stay as it was, it's convention to have it with the variable, I don't remember the reason, but please let it stay as it was.

Reply:
I read the documentation again and got more confused then before ;-)
https://docs.typo3.org/typo3cms/exceptions/master/en-us/Exceptions/1365429656.html

In ext_emconf.php you should use $_EXTKEY, but in ext_localconf.php you should use a hardcoded extension string...